scss/_global.scss

```diff
@@ -20,35 +20,50 @@ $global-width: rem-calc(1200) !default;
 /// @type Number
 $global-lineheight: 1.5 !default;
 
-/// Colors used for buttons, callouts, links, etc. There must always be a color called `primary`.
+/// Colors available by using `get-color()`.
```

Comment:
I mean: Please precise that unlike the previous get-color, these colors (`red`, `blue`) should not be used inside components, because component should be customizable and rely on any color.